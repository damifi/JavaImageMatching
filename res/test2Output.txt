./res/test2Bild0.bmp 170 146
./res/test2Bild1.bmp 259 611
./res/test2Bild2.bmp 368 280
./res/test2Bild3.bmp 812 622
7
