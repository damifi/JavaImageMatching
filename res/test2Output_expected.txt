./res/test2Bild0.bmp 170 146
./res/test2Bild1.bmp -1 -1
./res/test2Bild2.bmp -1 -1
./res/test2Bild3.bmp 812 622
31